src/instructlab/model/backends/backends.py

```diff
@@ -219,31 +219,34 @@ def ensure_server(
         # pylint: enable=duplicate-code
     except ClientException:
         port = free_tcp_ipv4_port(host)
-        logger.debug("Using %i", port)
+        logger.debug(f"Using available port {port} for temporary model serving.")
 
         host_port = f"{host}:{port}"
         temp_api_base = get_api_base(host_port)
-        logger.debug(f"Starting a temporary server at {temp_api_base}...")
         llama_cpp_server_process = None
         vllm_server_process = None
 
         if backend == VLLM:
             # TODO: resolve how the hostname is getting passed around the class and this function
             vllm_server_process = server_process_func(port)
+            logger.info(f"Starting a temporary vLLM server at {temp_api_base}...")
```

Comment:
```suggestion
            logger.info(f"Starting a temporary vLLM server at {temp_api_base}")
```
If we change that line I'd remove the `...` it's a bit weird to have this next to the API base URL I think.

Reply:
okay gotcha good recommendation